src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: 300ms;
```

Comment:
You should define CSS properties that changed on transition

Reply:
Thank you! fix it